dev-tools/mage/settings.go

```diff
@@ -41,9 +41,9 @@ const (
 	fpmVersion = "1.13.1"
 
 	// Docker images. See https://github.com/elastic/golang-crossbuild.
-	beatsFPMImage = "docker.elastic.co/beats-dev/fpm"
+	beatsFPMImage = "docker.elastic.co/observability-ci/fpm"
 	// BeatsCrossBuildImage is the image used for crossbuilding Beats.
-	BeatsCrossBuildImage = "docker.elastic.co/beats-dev/golang-crossbuild"
+	BeatsCrossBuildImage = "docker.elastic.co/observability-ci/golang-crossbuild"
```

Comment:
This won't be needed as soon as we merge https://github.com/elastic/golang-crossbuild/pull/527